apps/www/server/index.js

```diff
@@ -47,6 +48,34 @@ app.prepare().then(() => {
     }),
   )
 
+  if (process.env.DENY_IPS) {
+    try {
+      // We use JSON.parse because to filter a IP range it needs to be specified as an array, e.g. [['127.0.0.1', '127.0.0.10']]
+      const denyIPs = process.env.DENY_IPS
+        ? JSON.parse(process.env.DENY_IPS)
+        : []
+
+      console.log('Denying the following IPs:', denyIPs)
+
+      server.use(
+        ipfilter(denyIPs, {
+          mode: 'deny',
+          logLevel: 'deny',
+          detectIp: (req, res) => {
+            const ip = req.headers['x-forwarded-for']
+              ? req.headers['x-forwarded-for'].split(',')[0]
+              : req.connection.remoteAddress
+
+            return ip
+          },
```

Comment:
Built-in IP detected is likely more sane (uses [proxy-addr](https://www.npmjs.com/package/proxy-addr)). Would drop this.

In turn, if `!DEV`, set `trustProxy` to `true` since [it will run behind a proxy aka. Heroku Router](https://github.com/jetersen/express-ipfilter/blob/master/lib/ipfilter.js#LL38C7-L39C3).


```suggestion
          trustProxy: !DEV
```

Reply:
Ah yes, that should work. I just copied the example from the express-ipfilter README 🙄